### PR TITLE
Collision Monitor max_points to min_points rename

### PR DESCRIPTION
**Pull request by AlexeyMerzlyakov**
Related to the https://github.com/ros-planning/navigation2/pull/3513 `max_points` -> `min_points` parameter rename.
`max_points` parameter meant the maximum number of points where we are still not triggering the action, while `min_points` - is a number starting from the action to be initiated. In other words, `min_points = max_points + 1`. So, default parameter values were also updated.

### CODE REVIEW

File: configuration/packages/configuring-collision-monitor.rst [hunk 1]
```diff
@@ -393,14 +393,14 @@ For more information how to bring-up your own Collision Monitor node, please ref
           type: "circle"
           radius: 0.3
           action_type: "stop"
-          max_points: 3
+          min_points: 4
```
Comment (SteveMacenski) [inline configuration/packages/configuring-collision-monitor.rst:396]: Note about max_points for Humble

File: configuration/packages/configuring-collision-monitor.rst [hunk 2]
```diff
@@ -240,16 +240,16 @@ Polygons parameters
...
   ============== =============================
   Type           Default
   -------------- -----------------------------
-  int            3
+  int            4
   ============== =============================
 
   Description:
-    Maximum number of data readings within a zone to not trigger the action.
+    Minimum number of data readings within a zone to trigger the action.
```
[the remark this answers is not in the document]
Comment (AlexeyMerzlyakov) [reply configuration/packages/configuring-collision-monitor.rst]: Thanks for noting this item. Added reference to Humble in https://github.com/ros-planning/navigation.ros.org/pull/408/commits/472668524846e16644f4fd5f33514be5f7087260#diff-60e7d74687df0b5a07a3d91dc411c686e266df32edbdb653cf6e2ffc2674bc7cR252.

File: configuration/packages/configuring-collision-monitor.rst (hunk 1 above)
Comment (AlexeyMerzlyakov) [reply configuration/packages/configuring-collision-monitor.rst]: Done for parameters page and tutorial in https://github.com/ros-planning/navigation.ros.org/pull/408/commits/472668524846e16644f4fd5f33514be5f7087260

File: configuration/packages/configuring-collision-monitor.rst [hunk 3]
```diff
@@ -25,8 +25,8 @@ When multiple zones trigger at once, the most aggressive one is used (e.g. stop
 
 The following models of safety behaviors are employed by Collision Monitor:
 
-- **Stop model**: Define a zone and a point threshold. If more that ``max_points`` obstacle points appear inside this area, stop the robot until the obstacles will disappear.
-- **Slowdown model**: Define a zone around the robot and slow the maximum speed for a ``slowdown_ratio``, if more than ``max_points`` points will appear inside the area.
+- **Stop model**: Define a zone and a point threshold. If ``min_points`` or more obstacle points appear inside this area, stop the robot until the obstacles will disappear.
```
[the remark this answers is not in the document]
Comment (AlexeyMerzlyakov) [reply configuration/packages/configuring-collision-monitor.rst:28]: Added a sub-section line about the change in https://github.com/ros-planning/navigation.ros.org/pull/408/commits/472668524846e16644f4fd5f33514be5f7087260

**Review by SteveMacenski — APPROVED** [review]

**Pull request merged**